plugins/node/opentelemetry-instrumentation-restify/test/restify.test.ts

```diff
@@ -73,24 +87,29 @@ const getHandler: restify.RequestHandler = (req, res, next) => {
   res.send({ route: req?.params?.param });
 };
 const throwError: restify.RequestHandler = (req, res, next) => {
-  throw new Error('NOK');
+  throw new AppError('NOK');
+};
+const returnError: restify.RequestHandler = (req, res, next) => {
+  next(new AppError('NOK'));
 };
 
 const createServer = async (setupRoutes?: Function) => {
+  const restify = require('restify');
```

Comment:
This definition overruns the one above:
```js
import type * as restify from 'restify';
```

I wonder if the `restify` type import is used anywhere.
Can we `import * as restify from 'restify'` after we setup the instrumentation?

Reply:
The types are used at the top level: `useHandler`, etc. Import declarations can only be used at the top level so I cannot replace it as is.

The reason I changed it was to make sure the instrumentation is always at the correct state(enabled or disabled) before the require, but I guess I can bring it back to the top level, yes.

Will do.

Reply:
Done. PTAL, @blumamir